src/Stryker.Core/Stryker.Core/Stryker.Core.csproj

```diff
@@ -49,7 +53,6 @@
     <PackageReference Include="Microsoft.TestPlatform.ObjectModel" />
     <PackageReference Include="Microsoft.TestPlatform.Portable" />
     <PackageReference Include="Microsoft.TestPlatform.TranslationLayer" />
-    <PackageReference Include="Microsoft.Web.LibraryManager.Build" />
```

Comment:
This is necessary for the html report

Reply:
are you sure ? I mean it (obviously) build without it and I can generate html reports.
What is it necessary for?
(Asking because it seemed like a good idea to weed out some explicit dependencies that were not needed, for clarity, limit version conflicts and potentially improve start up performance)

Reply:
It executes libman during msbuild in the pipeline. We use this to download the static Javascript files that render the report. If you build using visual studio, vs will automatically run libman for you. Alternatively of you already have the files locally you also wouldn't notice that it's not working.

Reply:
thanks 
reverted. This is strange, I got the impression that the pipeline ran ok.
But now there is clearly a libman related error. Pfffff